apps/mobile/src/screens/onboarding/libs/components/content/content.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+	type Control,
+	type FieldErrors,
+	type FieldValues,
+} from "react-hook-form";
+
+import { RadioGroup, Text } from "~/libs/components/components";
+import { globalStyles } from "~/libs/styles/global-styles/global-styles";
+
+import {
+	type OnboardingAnswerDto,
+	type OnboardingFormValues,
+	type OnboardingQuestionResponseDto,
+	type RadioGroupValue,
+} from "../../types/types";
+import { styles } from "./styles";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T>;
+	errors: FieldErrors<T>;
+	question: null | OnboardingQuestionResponseDto;
+};
+
+const Content: React.FC<Properties<OnboardingFormValues>> = ({
+	control,
+	errors,
+	question,
+}) => {
+	const mapData: RadioGroupValue[] = question?.answers
```

Comment:
Let's give it some meaningful name, like 
```suggestion
	const mappedAnswersToRadioOptions: RadioGroupValue[] = question?.answers
```